app.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const jest_sorted = require('jest-sorted');
+const { getTopics } = require('./controllers/topics.controllers');
+
+const app = express();
+
+
+app.get('/api/topics', getTopics);
+app.all('*', (req, res) => {
+    res.status(404).send({ msg: 'error not found'})
+});
+
```

Comment:
Could you just add an app.use in here to pick up 500 errors please?

Reply:
app.use((err, req, res, next) => {
    res.status(500).send('Something is not working!');
});

Is that correct?